doc/scenarios.md

```diff
@@ -0,0 +1,39 @@
+# Alternative Scenarios
+
+This is a short primer for how to use some of the parameters available in the witan.send model in order to run _"alternative scenarios"_.
+
+Each of the paramater keys (e.g. `:splice-ncy`) can be found, and their values defined, in the [config.edn](https://github.com/MastodonC/witan.send/blob/master/data/demo/config.edn).
+
+More information on how to define a scenario can be found here (TBD). 
```

Comment:
This is currently [here](https://docs.google.com/document/d/1lQ2RrESpiyU5x2YUY8YvT287K4iq6NJw4jqHUIqC2VM/edit) (could a